skimage/restoration/_denoise.py

```diff
@@ -62,8 +66,19 @@ def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
     >>> astro = astro[220:300, 220:320]
     >>> noisy = astro + 0.6 * astro.std() * np.random.random(astro.shape)
     >>> noisy = np.clip(noisy, 0, 1)
-    >>> denoised = denoise_bilateral(noisy, sigma_range=0.05, sigma_spatial=15)
+    >>> denoised = denoise_bilateral(noisy, sigma_range=0.05,
+                                     sigma_spatial=15, multichannel=True)
```

Comment:
I don't know how to deal with this. The [doctests](https://travis-ci.org/scikit-image/scikit-image/jobs/104567193#L2938) fail here. I really don't want to put 100 columns in a line.